pkg/interface/chat/src/js/components/lib/message.js

```diff
@@ -71,7 +71,7 @@ export class Message extends Component {
           ></img>
         );
         return (
```

Comment:
@matildepark my understanding of the code is that it, if imgMatch, it returns the the classes in className. It seems to render that correctly?

![image](https://user-images.githubusercontent.com/9557907/73203953-f2eb6d80-40f2-11ea-9ba3-0133f2a94265.png)